test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Xml.Tests.ps1

```diff
@@ -59,22 +59,21 @@ Describe "ConvertTo-Xml DRT Unit Tests" -Tags "CI" {
     }
 
     It "StopProcessing should work" {
-		$ps = [PowerShell]::Create()
-		$ps.AddCommand("Get-Process")
-		$ps.AddCommand("ConvertTo-Xml")
-		$ps.AddParameter("Depth", 2)
-		$ps.BeginInvoke()
-		$ps.Stop()
-		$ps.InvocationStateInfo.State | Should -BeExactly "Stopped"
+        $ps = [PowerShell]::Create()
+        $null = $ps.AddScript({ Get-Process | ConvertTo-Xml -Depth 2 -Verbose })
+        $null = $ps.BeginInvoke()
+        Wait-UntilTrue { $ps.Streams.Verbose.Count -gt 0 } -IntervalInMilliseconds 50
+        $null = $ps.Stop()
+        $ps.InvocationStateInfo.State | Should -BeExactly "Stopped"
```

Comment:
these is no guarantee that this will take a significant about of time.  It may finish before you stop it.

Reply:
@JamesWTruher  Can you review?

Reply:
> these is no guarantee 

Yes, I'd remove the tests - I do not see a way to make these tests reliable. If we just add a hook with an infinite loop...

Reply:
The intent of the test is to cover the StopProcessing code path as identified by code coverage.  I wonder if we can just have at test hook in the cmdlet so that ProcessRecord() doesn't return if the test hook is in place.  Therefore we can call `Stop()` so that the StopProcessing code path gets exercised?

Reply:
Seems like the implementation would be for `ProcessRecord` to Sleep for enough time that we are certain enough that we can stop it.  This could also mean that we can get one object and process it instead of trying to process a lot of records trying to make it take a long time.

Reply:
Test hook added.
Sleep suspend current thread. So I used only short sleep to slow down a cmdlet.